iothub/service/src/Registry/Models/ImportJobProperties.cs

```diff
@@ -159,46 +125,29 @@ public JobProperties(Uri outputBlobContainerUri, bool excludeKeysInExport)
         [JsonProperty(PropertyName = "inputBlobName", NullValueHandling = NullValueHandling.Ignore)]
         public string InputBlobName { get; set; }
 
-        /// <summary>
-        /// The name of the blob that will be created in the provided output blob container. This blob will contain
-        /// the exported device registry information for the IoT hub.
-        /// </summary>
-        /// <remarks>
-        /// If not specified, defaults to "devices.txt".
-        /// </remarks>
-        [JsonProperty(PropertyName = "outputBlobName", NullValueHandling = NullValueHandling.Ignore)]
-        public string OutputBlobName { get; set; }
-
-        /// <summary>
-        /// Optional for export jobs; ignored for other jobs. Default: false. If false, authorization keys are included
-        /// in export output. Keys are exported as null otherwise.
-        /// </summary>
-        [JsonProperty(PropertyName = "excludeKeysInExport", NullValueHandling = NullValueHandling.Ignore)]
-        public bool ExcludeKeysInExport { get; set; }
-
         /// <summary>
         /// Specifies authentication type being used for connecting to storage account.
         /// </summary>
         [JsonProperty(PropertyName = "storageAuthenticationType", NullValueHandling = NullValueHandling.Ignore)]
         public StorageAuthenticationType? StorageAuthenticationType { get; set; }
 
         /// <summary>
-        /// The managed identity used to access the storage account for import and export jobs.
+        /// The managed identity used to access the storage account for import job.
         /// </summary>
         [JsonProperty(PropertyName = "identity", NullValueHandling = NullValueHandling.Ignore)]
         public ManagedIdentity Identity { get; set; }
 
         /// <summary>
-        /// Whether or not to include configurations in the import or export job.
+        /// Whether or not to include configurations in the import job.
         /// </summary>
         /// <remarks>
-        /// The service assumes this is false, if not specified. If true, then configurations are included in the data export/import.
```

Comment:
extreme nit: `import/export` not `export/import`

Reply:
ignoring as the line/comment was removed.